cmake/Findquatlib.cmake

```diff
@@ -65,19 +65,35 @@ else()
 		/usr/local)
 
 	# Look for the library.
-	find_library(QUATLIB_LIBRARY
+	find_library(QUATLIB_LIBRARY_RELEASE
 		NAMES
 		quat.lib
 		libquat.a
-		HINTS
+		PATH_SUFFIXES
+		ReleaseAcademicEdition
+		Release
+		${_libsuffixes}
+		PATHS
 		"${QUATLIB_ROOT_DIR}"
+		"${_progfiles}/VRPN"
+		"${_progfiles}/quatlib"
+		C:/usr/local
+		/usr/local)
+	find_library(QUATLIB_LIBRARY_DEBUG
+		NAMES
+		quat.lib
+		libquat.a
 		PATH_SUFFIXES
+		DebugAcademicEdition
+		Debug
 		${_libsuffixes}
 		PATHS
+		"${QUATLIB_ROOT_DIR}"
 		"${_progfiles}/VRPN"
 		"${_progfiles}/quatlib"
 		C:/usr/local
 		/usr/local)
+	select_library_configurations(QUATLIB)
```

Comment:
Was going to say, I think the capitalization is wrong, but double-checked and nope, you got it totally right here - it's the `find_package_handle_standard_args` one that has the commonly-confused capitalization (where it needs the name of the package, not the prefix of the variables - it will capitalize on its own by default).

Nice work! 👍 😁